### PR TITLE
PAN-OS IP arg and support job_id as array

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19680

## Related PRs
pending on
https://github.com/demisto/content/pull/4807

@David-BMS FYI.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - soon
- [ ] Documentation - soon
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file - integration](https://github.com/demisto/content/blob/e1d82dd67485081d4e93007379575c95d280a87b/Integrations/Panorama/CHANGELOG.md)
- [ ] [YAML file - playbook](https://github.com/demisto/content/blob/8e108b5d0ff587a0b8e16ee4fb8a4f6beb1ccda7/Playbooks/playbook-Panorama_Query_Logs_CHANGELOG.md)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/e1d82dd67485081d4e93007379575c95d280a87b/Integrations/Panorama/Panorama.yml)

### CODE REVIEW

**Comment by bakatzir** [comment]
fix: https://github.com/demisto/etc/issues/18773

**Comment by bakatzir** [comment]
@kirbles19 All relevant files are linked in the main comment.

File: Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -2558,6 +2558,12 @@ script:
       name: addr-dst
       required: false
       secret: false
+    - default: false
+      name: ip
+      description: Destination or Source address.
```
Comment (kirbles19) [inline Integrations/Panorama/Panorama.yml]: Destination or source IP address.

File: Playbooks/playbook-Panorama_Query_Logs.yml [hunk 2]
```diff
@@ -380,6 +383,10 @@ inputs:
   value: {}
   required: false
   description: Destination address.
+- key: ip
+  value: {}
+  required: false
+  description: Source or Destination address.
```
Comment (kirbles19) [inline Playbooks/playbook-Panorama_Query_Logs.yml]: Source or destination IP address.

**Comment by bakatzir** [comment]
Updated the branch from master, please add the docs-approved label if done.

**Comment by yuvalbenshalom** [comment]
@kirbles19 @yaron-libman please add docs approved or explain what is missing.

**Pull request merged**